pkg/core/namespace.go

```diff
@@ -178,6 +178,9 @@ func (c *kubectlClient) NamespaceInit(manifests map[string]*Manifest, options Na
 
 		fmt.Printf("Applying %s in namespace %q\n", release, ns)
 		resourceUrl, _ := url.Parse(resource)
+		if resourceUrl.Scheme == "" {
+			resourceUrl.Scheme = "file"
```

Comment:
If `path` does not begin with a "slash" (TBD windows), the url needs to be made absolute. All URLs with scheme == file must use absolute paths

Reply:
Understood. `resource` was returned from `fileutils.IsAbsFile` and so will be absolute - see the godoc of that function for details.

Reply:
Ah, sorry, my bad. Did a quick test with just `url.Parse()` and saw that it could accept relative paths.
Please disregard then